ckanext/switzerland/helpers/date_helpers.py

```diff
@@ -100,3 +100,13 @@ def display_if_datetime(value):
             return isodate.strftime(value, DATE_FORMAT)
     except Exception:
         return None
+
+
+def is_null_date_value(value):
+    """date values stored in postgres as not set"""
+    if value in ['', 'False']:
+        return True
+
+
+class OGDCHDateValidationException(Exception):
```

Comment:
Fo the case when value = False: Could we do this Exception not silently? Perhaps, it would be useful then later to find out why some time values are stored as `False`

Reply:
You mean I should log the exception? Because currently our test data dump contains all those values. And the code needs to work with it. So I think for now I will leave it like this. Okay for you?

Reply:
to log the exception sounds good to me, (in general ) would be interesting to see how often the date with value `False`